src/app/core/components/storage-dialog/storage-dialog.component.html

```diff
@@ -61,6 +61,9 @@
             >
           </p>
           <pr-new-pledge [inlineFlow]="true"></pr-new-pledge>
+          <button class="btn btn-link redeem-promo" (click)="setTab('promo')">
+            Have a code to redeem for free storage? Click here to redeem it.
+          </button>
```

Comment:
We might want this to be an `a` tag instead since from the user perspective, this is essentially a link. For accessibility purposes we probably want this to be a link to match that expectation rather than it being a button.